components/style/atomic_refcell.rs

```diff
@@ -2,7 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#![allow(unsafe_code)]
```

Comment:
I think you can put it individually above each `unsafe impl` instead. I think that given that the amount of unsafe code that exists here is really low that'd be preferrable.